tests/phpunit/tests/kses.php

```diff
@@ -1991,4 +1991,28 @@ public function data_kses_globals_are_defined() {
 			),
 		);
 	}
+
+	/**
+	 * @ticket 56266
+	 *
+	 * @covers ::wp_filter_global_styles_post
+	 */
+	public function test_wp_filter_global_styles_post_returns_correct_value() {
+		$user_theme_json = '{
+ 			"isGlobalStylesUserThemeJSON": 1,
+ 			"version": 1,
+ 			"settings": {
+ 				"typography": {
+ 					"fontFamilies": {
+ 						"fontFamily": "\"DM Sans\", sans-serif",
+ 						"slug": "dm-sans",
+ 						"name": "DM Sans",
+ 					}
+ 				}
+ 			}
+ 		}';
+
+		$filtered_user_theme_json = wp_filter_global_styles_post( $user_theme_json );
+		$this->assertEquals( $user_theme_json, $filtered_user_theme_json, 'Filtered and expected json data must match.' );
```

Comment:
Use `assertSame()` instead of `assertEquals()`.

Reply:
As there are no other tests for this function, I'd also suggest moving this function's test to its own test class: `tests/phpunit/tests/kses/wpFilterGlobalStylesPost.php`. In doing so, it will kick-off the work to separate each kses function into its own test class.

Reply:
Fixed in ee805ff646fbe5df325ff6c57ab2a0c99ce26761

Reply:
Good point. Fixed in 895f6cfdd799ee50133e2075d506c464f9b5cda2